### PR TITLE
Added intro slides to website.

### PR DESCRIPTION
**Pull request by jrtechs**
I added the first-week meeting slide deck to the website.

Screenshot from 2019-08-31 11-16-39

### CODE REVIEW

File: talks/_posts/2019-08-30-welcome-to-ritlug.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Fall 2019 - Welcome to RITlug!
+author: Jeffery Russell
+layout: post
+slides: 2019-w01-fall-into-to-linux.pdf
```
Comment (Tjzabel) [inline talks/_posts/2019-08-30-welcome-to-ritlug.md]: *intro* to Linux.

Small typo here

File: talks/_posts/2019-08-30-welcome-to-ritlug.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+date: 2019-08-30
+---
+
+Welcome to another year of the RIT Linux Users Group, fall 2016 edition!
```
Comment (Tjzabel) [inline talks/_posts/2019-08-30-welcome-to-ritlug.md]: What do you mean by **2016** edition?

**Comment by Tjzabel** [comment]
@jrtechs I also updated this branch, so make sure you do a pull/rebase with master before you try to push further changes

File: talks/_posts/2019-08-30-welcome-to-ritlug.md (hunk 1 above)
Comment (jrtechs) [reply talks/_posts/2019-08-30-welcome-to-ritlug.md]: lol, I can always count on you to fix my spelling :smile:

File: talks/_posts/2019-08-30-welcome-to-ritlug.md (hunk 2 above)
Comment (jrtechs) [reply talks/_posts/2019-08-30-welcome-to-ritlug.md]: dope, I think I just copied in the lingo from the 2016 intro slide post

**Comment by jrtechs** [comment]
Cool, I just made those updates @Tjzabel

**Pull request merged**